lib/active_support/cache/redis_store.rb

```diff
@@ -310,6 +317,22 @@ def normalize_key(*args)
             namespaced_key(*args)
           end
         end
+
+        def failsafe(method, returning: nil)
+          yield
+        rescue ::Redis::BaseConnectionError => e
```

Comment:
Now `ERRORS_TO_RESCUE` is not in use - shouldn't this method use that or `ERRORS_TO_RESCUE` should be removed.

Reply:
Why not just create a PR to remove it?